setuptools/dist.py

```diff
@@ -280,7 +280,9 @@ def check_requirements(dist, attr, value):
             "{attr!r} must be a string or list of strings "
             "containing valid project/version requirement specifiers; {error}"
         )
-        raise DistutilsSetupError(tmpl.format(attr=attr, error=error))
+        raise DistutilsSetupError(
+            tmpl.format(attr=attr, error=error)
```

Comment:
Note to fix later: with this change, the embedding of `error` is no longer necessary.